aten/src/ATen/native/transformers/cuda/mem_eff_attention/gemm_kernel_utils.h

```diff
@@ -162,7 +162,7 @@ struct DefaultGemmType<
         cutlass::sizeof_bits<scalar_t>::value == 16>::type> {
   static constexpr int ThreadK = 32;
   static constexpr int WarpK = 32;
-  static constexpr int kMinimumAlignment = 4;
+  static constexpr int kMinimumAlignment = 8;
```

Comment:
is the minimum alignment 8, or is it 4 for FP32 and 8 for FP16 (i.e., really 128b ?)

Reply:
AFAI this is templated and this template refers to 16 bit types so 8 for this case.